README.md

```diff
@@ -55,7 +55,7 @@ west update
 To build the application, run the following command:
 
 ```shell
-west build -b $BOARD app
+west build -b $BOARD ncs-example-application/app
```

Comment:
this is not correct, user is expected to execute from the repository root

Reply:
Where is the expectation stated? It seems that README does not state that user is required to have previous knowledge on building Zephyr products; if user follows the README, building is not possible.

Reply:
@de-nordic

Reply:
then we need to add an extra `cd`, or modify `cd my-workspace` to `cd my-workspace/app`

Reply:
User is already at my-workspace from previous step, but app is not. app is at lower level which is ncs-example-application.
I went through the not modified procedure and have recreated the dir structure.